install/pre-install-payload/Dockerfile

```diff
@@ -1,25 +1,47 @@
-FROM ubuntu:22.04
+#### Confidential Containers forked containerd
+
+FROM alpine:3.18 as coco-containerd-binary-downloader
```

Comment:
Just a nit: instead of repeat `FROM alpine:3.18` for the three images, you can create a `FROM alpine:3.18 as base`  and the subsequent images takes `FROM base`. On that base image you can even update the image like `apt get update` (don't know if Alpine has the same update mechanism)

Reply:
Perfect, will do!